package.json

```diff
@@ -25,22 +27,25 @@
     "prepare": "patch-package",
     "prettier-project": "yarn run prettier-write 'src/**/*.{ts,tsx,js,jsx}'",
     "prettier-write": "yarn run prettier --write",
-    "publish-assets:local": "yarn assets && yarn publish-assets",
     "publish-assets": "node scripts/uploadToS3.js",
+    "publish-assets:local": "yarn clean && yarn build:assets && yarn publish-assets",
     "relay": "relay-compiler",
     "sentry": "scripts/sentry.sh",
-    "start:dev": "NODE_ENV=development BUILD_CLIENT=true node --inspect --max_old_space_size=3072 -r dotenv/config ./src",
-    "start:prod:local": "NODE_ENV=production node --inspect --max_old_space_size=3072 -r dotenv/config ./server.dist.js",
-    "start:prod": "yarn assets && yarn build:server && NODE_ENV=production yarn start",
     "start": "scripts/start.sh",
+    "start:dev": "concurrently 'yarn start:dev:relay' 'yarn start:dev:server'",
+    "start:dev:server": "NODE_ENV=development BUILD_CLIENT=true node --inspect --max_old_space_size=3072 -r dotenv/config ./src/index.dev.js",
```

Comment:
Why is this needed? How is it separate from other commands such as `yarn start` (which product devs will use) or `yarn start:dev:server`?